api/v1beta1/common.go

```diff
@@ -127,11 +127,15 @@ type AnsibleEESpec struct {
 	// AnsibleSkipTags for ansible execution
 	AnsibleSkipTags string `json:"ansibleSkipTags,omitempty"`
 	// ExtraVars for ansible execution
-	ExtraVars map[string]json.RawMessage  `json:"extraVars,omitempty"`
+	ExtraVars map[string]json.RawMessage `json:"extraVars,omitempty"`
 	// ExtraMounts containing files which can be mounted into an Ansible Execution Pod
 	ExtraMounts []storage.VolMounts `json:"extraMounts,omitempty"`
 	// Env is a list containing the environment variables to pass to the pod
 	Env []corev1.EnvVar `json:"env,omitempty"`
 	// DNSConfig for setting dnsservers
 	DNSConfig *corev1.PodDNSConfig `json:"dnsConfig,omitempty"`
+	// ServiceAccountName allows to specify what ServiceAccountName do we want
+	// the ansible execution run with. Without specifying, it will run with
+	// default serviceaccount
+	ServiceAccountName string
```

Comment:
```suggestion
	ServiceAccountName string `json:"serviceAccountName,omitempty"`
```

Reply:
why are json tags needed on the field of this struct? The struct is not part of the CRD. I see they're on the other fields, but I was wondering if they're actually needed.

Reply:
I didn't realize it wasn't in the CRD part